### PR TITLE
json error reponse for all errors and successes

### PR DESCRIPTION
**Pull request by SahSantoshh**
I want you to add missing things on TODOs @sanzog03

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -44,11 +45,20 @@ const userRoutes = require("./routes/user.route");
...
+  return res.status(200).json({ status: 'success', statusCode: 200, message: "Greetings from nsa api" });
 });
 
 app.use("/api/user", userRoutes);
 
+// an error route
+app.get('/error', (req, res) => {
+  throw new ErrorHandler(500, 'Internal server error');
+});
+
```
Comment (sanzog03) [inline app.js:57]: why is this route necessary?? will anybody do `ip:port/error` routes. i dont see its purpose.

Comment (sanzog03) [reply app.js:57]: get request on `ip:port/error` doesn't seem right

File: app.js [hunk 2]
```diff
@@ -44,11 +45,20 @@ const userRoutes = require("./routes/user.route");
...
+app.use((err, req, res, next) => {
+  handleError(err, res);
+});
```
Comment (sanzog03) [inline app.js:60]: i believe next() call should be added here

Comment (sanzog03) [reply app.js:60]: after handleError()

File: app.js (hunk 1 above)
Comment (SahSantoshh) [reply app.js:57]: Got it will remove it.

File: app.js (hunk 2 above)
Comment (SahSantoshh) [reply app.js:60]: Can you add an example?

Comment (SahSantoshh) [reply app.js:60]: is it something like this.
`app.use((err, req, res, next) => next(), handleError(err, res));
`

**Pull request merged**